keystore/src/entities/mod.rs

```diff
@@ -172,6 +178,17 @@ cfg_if::cfg_if! {
             id: Vec<u8>,
         }
 
+        #[async_trait::async_trait(?Send)]
+        pub trait EntityMlsExt: Entity {
+            async fn mls_delete<'a>(tx: &crate::connection::storage::WasmStorageTransaction<'a>, id: StringEntityId<'a>) -> CryptoKeystoreResult<()> {
+                tx.delete(Self::COLLECTION_NAME, id.as_slice()).await
+            }
+            async fn mls_save<'a>(&'a self, tx: &crate::connection::storage::WasmStorageTransaction<'a>) -> CryptoKeystoreResult<()>;
```

Comment:
Can we provide a default implementation here that will be used in most or all entities to avoid some code duplication?

Reply:
For this specific one unfortunately no. I've tried. The compiler can't figure out the type for the second parameter.

Reply:
Actually I've managed to make it work. Fixed in: https://github.com/wireapp/core-crypto/pull/695/commits/af6abef688cd89ab886d6aee49cb1f089c7dc3ba

Reply:
Nice! Now to avoid repeating an empty impl block for every entity you can, AFAICT, do instead once 
```rust
impl<T: Entity> EntityMlsExt for T {}
```

Reply:
Good point. I'll try it out

Reply:
The problem is the few ones that just throw errors... what should we do in this case? We can't have it implementing the same struct twice.

Reply:
Hmm, right... I guess let's leave it with the repition for now, the refactor needed for that is out of scope then 

This throwing errors is such a design flaw. If they only throw errors the type shouldn't claim that it implements the trait :)